README.md

```diff
@@ -1,5 +1,7 @@
 # [Gardener Extensions](https://gardener.cloud)
 
+<img src="https://user-images.githubusercontent.com/5526658/65959704-f9265180-e452-11e9-8f5e-5f2038128915.png" width="180"/>
```

Comment:
I guess you could add it under /logo/ as in gardener/gardener.

Reply:
ok will do

Reply:
done @ialidzhikov